### PR TITLE
[docs] add note about build:android

### PR DESCRIPTION
**Pull request by gregfenton**
# Why

Following the instructions as-is with a new project leads to an impasse and a cryptic message.

# How

Add note & link about running the Android build before using `expo fetch:android:hashes`

# Test Plan

(docs review)

### CODE REVIEW

File: docs/pages/versions/v38.0.0/sdk/map-view.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ If you have already integrated Google Sign In into your standalone app, this is
   3.  In `app.json`, copy the API key from `android.config.googleSignIn` to `android.config.googleMaps.apiKey`.
   4.  Rebuild your standalone app.
 - **If you already have not configured Google Sign In**
-  1.  Build your app, take note of your Android package name (eg: `ca.brentvatne.growlerprowler`)
+  1.  [Build your app for Android](https://docs.expo.io/distribution/building-standalone-apps/#if-you-choose-to-build-for-android) with `expo build:android`.  Take note of your Android package name (eg: `ca.brentvatne.growlerprowler`)
```
Comment (brentvatne) [inline docs/pages/versions/v38.0.0/sdk/map-view.md:67]:
```suggestion
  1.  [Build your app for Android](/distribution/building-standalone-apps/#if-you-choose-to-build-for-android) with `expo build:android`. Take note of your [Android package name](https://expo.fyi/android-package).
```

**Comment by brentvatne** [comment]
make some small changes. can you apply this to map-view doc in the unversioned directory too? thanks!

**Comment by gregfenton** [comment]
Same change made to unversioned.

There are other references to `ca.brentvatne.growlerprowler` on that page (one Android, one iOS).  Should they all be replaced with the FYI link?

**Comment by brentvatne** [comment]
it looks like this fell between the cracks. i'll close it but if this still applies just ping me on discord and we can figure out a patch

**Pull request closed without merging**